charger/keba-modbus.go

```diff
@@ -265,6 +300,11 @@ var _ api.ChargerEx = (*Keba)(nil)
 func (wb *Keba) MaxCurrentMillis(current float64) error {
 	u := uint16(current * 1000)
 	_, err := wb.conn.WriteSingleRegister(kebaRegMaxCurrent, u)
```

Comment:
as an evcc noob/novice, would some other evcc logic not just re-enable the charging session when the maxCurrent is being set somewhere else? 

i mean, that may not be all that bad, but still - previously i imagine the wallbox would have just ignored a new maxCurrent until a dedicated enable(true) came along. now on the P40, enablement would also happen by  adjusting the maxcurrent to a valid range value. 

problem?

Reply:
I'm new to evcc myself, so it's quite possible I'm missing something. If this was a problem, there might be a bigger problem as there are also other chargers that implement this in the same way, e.g. https://github.com/evcc-io/evcc/blob/master/charger/innogy.go#L131.

Reply:
True. Thx for this PR, definitely looks cleaner than my first go tryouts 🤣 
will test/use as soon as my car arrives